packages/did/src/DidDetails/LightDidDetails.ts

```diff
@@ -53,7 +87,13 @@ export class LightDidDetails extends DidDetails {
       did = did.concat(':', encodedDetails)
     }
 
-    super(did, id)
+    super(
+      did,
+      id,
+      serviceEndpoints.map((service) => {
+        return { ...service, id: `${did}#${service.id}` }
```

Comment:
Can you not use the `assembleKeyId`?

Reply:
But this is not a key, it is a service endpoint id.

Reply:
Again, this should be made into a utility function.